odoo/addons/base/models/ir_fields.py

```diff
@@ -200,6 +200,16 @@ def _str_to_json(self, model, field, value):
             msg = _("'%s' does not seem to be a valid JSON for field '%%(field)s'")
             raise self._format_import_error(ValueError, msg, value)
 
+    def _str_to_properties(self, model, field, value):
```

Comment:
is this robust ? (e.g. for many2one, etc)

shouldn't we just raise an clean error message ?

Reply:
Yes it is working for all many2one, many2many, etc.

Reply:
because the ids are the same right ? if you import in a different db with different ids it won't work no ?

Reply:
> because the ids are the same right ? if you import in a different db with different ids it won't work no ?

Yes, it won't work if we import in a different db with different ids.

Reply:
with many2one we export the display name if I'm not wrong, so either we should do the same or we should not allow to import properties imo

Reply:
> with many2one we export the display name if I'm not wrong, so either we should do the same or we should not allow to import properties imo

Changes are done!!